dissect/target/plugins/os/unix/sshd.py

```diff
@@ -0,0 +1,160 @@
+import logging
```

Comment:
This is a very interesting contribution, thank you.

We very much appreciate this contribution and all the effort but this potential new plugin has sparked an important internal debate in our development team about the future of configuration plugins on UNIX and Linux. We would like to propose to keep this plugin and potential future plugins regarding  UNIX configuration **on-hold** for now, just to prevent you from wasting any of your valuable time.

Please allow us some time to reflect on our current strategy before we proceed.

Reply:
To clarify, we are not rejecting this plugin but we are researching if there can be a more generic approach to querying configurations for Unix/Linux systems and how to integrate your contributions in this regard properly. But before we can be more clear about it, we need to have a better idea of how to approach this architectural challenge.